lib/libutils/isoc/bget_malloc.c

```diff
@@ -824,6 +833,14 @@ void *memalign(size_t alignment, size_t size)
 	return p;
 }
 
+#if __STDC_VERSION__ >= 201112L
+void *aligned_alloc(size_t alignment, size_t size)
+{
+	assert(!(size % alignment));
```

Comment:
No, in this case NULL should be returned (and `errno` set to `EINVAL` if we had an `errno` variable, but we don't :wink:)

Reply:
ok. done (but for `errno`). Thanks.